lib/flipper/action_patch.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module FlipperExtensions
+  module ActionPatch
+    def view(name)
+      # Use custom views if enabled in configuration.
+      path = custom_views_path.join("#{name}.erb") unless custom_views_path.nil?
+
+      # Fall back to default views if custom views haven't been enabled
+      # or if the custom view cannot be found.
+      path = views_path.join("#{name}.erb") if path.nil? || !path.exist?
+
+      raise "Template does not exist: #{path}" unless path.exist?
+
+      contents = path.read
+      compiled = Flipper::UI::Eruby.new(contents)
+      compiled.result proc {}.binding
+    end
+
+    def custom_views_path
+      Flipper::UI.configuration.custom_views_path
+    end
+
+    # This is where we store the feature descriptions.
+    # You can choose to store this where it makes sense for you.
+    def yaml_features
+      @yaml_features ||= FLIPPER_FEATURE_CONFIG['features']
+    end
```

Comment:
Pulled this in from the previous monkey patch so it's available to the other views in the Flipper UI.